content/en/docs/troubleshooting/_index.md

```diff
@@ -199,3 +199,89 @@ Use the following command to list the events in chronological order:
 ```bash
 {{% param cliToolName %}} get events --sort-by=.metadata.creationTimestamp --namespace <namespace>
 ```
+
+
+## Dry-run
+
+To help verify changes, you can use the optional `{{% param cliToolName %}}` flag `--dry-run=client -o yaml` to see the rendered YAML definition of your Kubernetes objects, without sending it to the API.
+
+The following `{{% param cliToolName %}}` subcommands support this flag (non-final list):
+
+* `apply`
+* `create`
+* `expose`
+* `patch`
+* `replace`
+* `run`
+* `set`
+
+For example, we can use the `--dry-run=client` flag to create a template for our Deployment:
+
+```bash
+{{% param cliToolName %}} create deployment example-web-go --image=quay.io/acend/example-web-go:latest --namespace acend-test --dry-run=client -o yaml
+```
+
+The result is the following YAML output:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: example-web-go
+  name: example-web-go
+  namespace: acend-test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: example-web-go
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: example-web-go
+    spec:
+      containers:
+      - image: quay.io/acend/example-web-go:latest
+        name: example-web-go
+        resources: {}
+status: {}
+```
+
+
+## `{{% param cliToolName %}}` API requests
+
+If you want to see the HTTP requests `{{% param cliToolName %}}` sends to the Kubernetes API in detail, you can use the optional flag `--v=10`.
+
+For example, to see the API request for creating a namespace:
+
+```bash
+{{% param cliToolName %}} create namespace acend-test --v=10
```

Comment:
This will produce a permission error in our Webshell / K8S environment as the ServiceAccount is not allowed to create new namespaces.
Alt ought its an example, people might wan't to execute it by themself

Reply:
Thanks for the hint, I'll create a new PR.

Reply:
#478